lib/check-tags.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const log = require('./out')({});
```

Comment:
I think it's the first file outside of `bin` that directly requires this file. Shouldn't it receive the context and use its `data` event?
In that case, my previous comment becomes obsolete.